packages/backend/src/graphql/resolvers/walletAddressKey.ts

```diff
@@ -4,10 +4,46 @@ import {
   WalletAddressKey as SchemaWalletAddressKey,
   Alg,
   Kty,
-  Crv
+  Crv,
+  WalletAddressResolvers
 } from '../generated/graphql'
 import { ApolloContext } from '../../app'
 import { WalletAddressKey } from '../../open_payments/wallet_address/key/model'
+import { Pagination } from '../../shared/baseModel'
+import { getPageInfo } from '../../shared/pagination'
+
+export const getWalletAddressKeys: WalletAddressResolvers<ApolloContext>['walletAddressKeys'] =
+  async (
+    parent,
+    args,
+    ctx
+  ): Promise<ResolversTypes['WalletAddressKeyConnection']> => {
+    const walletAddressKeyService = await ctx.container.use(
+      'walletAddressKeyService'
+    )
+
+    const getPageFn = (pagination_: Pagination) => {
+      if (!parent.id) throw new Error('missing wallet address id')
+      return walletAddressKeyService.getPage(parent.id, pagination_)
+    }
+
+    const { ...pagination } = args
+
+    const walletAddressKeys = await getPageFn(pagination)
+    ctx.logger.info({ walletAddressKeys }, 'got walletAddressKeys')
```

Comment:
```suggestion
```

Reply:
@tadejgolobic would you recommend applying this logging pattern more generally to other resources? in our http routes/gql resolvers? 

I know Max is already working on stuff, but at that point I wonder about simply adding a logging middleware that logs the response (and request for the matter). Although perhaps this is upstream enough from that that doing it here is still useful too.

Reply:
For me, this was actually a debug log, however, it wouldn't hurt to have this kind of logs. So basically on every request, to log URL and any parameters / body that will be passed and ofc response. This would show only if the log level is set to debug.

However, we should probably make sure, that we are not exposing some sensitive data in the log. This is probably the hardest thing to do

Reply:
> For me, this was actually a debug log, however, it wouldn't hurt to have this kind of logs. So basically on every request, to log URL and any parameters / body that will be passed and ofc response. This would show only if the log level is set to debug.

I'm currently working on this.